packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -114,18 +122,81 @@ const Watchlist = () => {
     setWatchlist(watchlist.filter(item => item !== symbol));
   };
 
+  const getWindowOptions = async () => {
+    const targetApplication = await OpenfinApiHelpers.getStockFluxApp(
+      'stockflux-chart'
+    );
+    const manifestContents = await fetch(targetApplication.manifest, {
+      method: 'GET'
+    });
+
+    const info = await manifestContents.json();
+    return info.startup_app;
+  };
+
+  useEffect(() => {
+    getWindowOptions().then(value => {
+      setPreviewOptions(value);
+    });
+  }, []);
+
+  useEffect(() => {
+    const windowOffset = 5;
```

Comment:
What is 5? Also maybe we should declare this const using `UPPER_CASE_SNAKE_CASE`?

Reply:
A random small number to make sure there is a gap between the 2 windows, making them clearer to see.
Changed the constant to use the correct case.